backend/tests/clubs/test_views.py

```diff
@@ -1044,20 +1046,22 @@ def test_club_create_url_sanitize(self):
 
         exploit_string = "javascript:alert(1)"
 
-        resp = self.client.post(
-            reverse("clubs-list"),
-            {
-                "name": "Bad Club",
-                "tags": [],
-                "facebook": exploit_string,
-                "twitter": exploit_string,
-                "instagram": exploit_string,
-                "website": exploit_string,
-                "linkedin": exploit_string,
-                "github": exploit_string,
-            },
-            content_type="application/json",
-        )
+        with patch("django.conf.settings.QUEUE_OPEN", True):
+            with patch("django.conf.settings.NEW_QUEUE_OPEN", True):
```

Comment:
Nit: can throw both these patches under the same context manager. Same goes for all the ones below.